rl_zoo3/exp_manager.py

```diff
@@ -344,7 +344,7 @@ def read_hyperparameters(self) -> tuple[dict[str, Any], dict[str, Any]]:
         else:
             raise ValueError(f"Hyperparameters not found for {self.algo}-{self.env_name.gym_id} in {self.config}")
 
-        if self.storage and self.study_name and self.trial_id:
+        if self.storage and self.study_name:
```

Comment:
I think this check is needed (see failures on CI), instead we could allow passing a special value like `-1` to allow loading the best trial